index.html

```diff
@@ -37,5 +37,174 @@ <h3 class="hey">Hey there <br> I'm Shadab</h3>
             <li class="fa fa-youtube fa-2x"><a href="#"></a></li>            
         </ul>
     </section>
+    <section class="works">
+        <div class="recentworkContainer">
+            <h1 class="recentWork">My Recent Work</h1>
+            <div class="line"></div>
+            <img src="images/recentWork.png" alt="the photo of my recent project">
+            <h3 class="recentWorkTitle">Multi-Post Stories</h3>
+            <p class="recentWorkDesc">A daily selection of privately personalized reads; no accounts or 
+                sign-ups required. has been the industry's standard dummy text ever since the 1500s, when an unknown 
+                printer took a standard dummy text.</p>
+            <ul class="usedTechs">
+                <li class="tech">CSS</li>
+                <li class="tech">HTML</li>
+                <li class="tech">Bootstrap</li>
+                <li class="tech">Ruby</li>
+            </ul>
+            <div class="seeProject">
+                <a href="#">See Project</a>
+            </div>
+        </div>
+        <div class="workCard">
+            <div class="workCardContent">
+                <h3 class="recentWorkTitle wc">Profesional Art <br>Printing Data</h3>
```

Comment:
- [ ] The works section seems to have some issues, it is not well aligned compared to the Figma design, it is stack to the right because it does not have margin, seen the deference below 👇 
![image](https://user-images.githubusercontent.com/55465133/129987307-e2abd013-71b7-491c-a983-832a501f89fb.png)

The Figma design
![image](https://user-images.githubusercontent.com/55465133/129987349-e4e81a12-8f68-4151-a5a6-03127b9b7f42.png)

Can you see the difference? Please let's fix this issue 👍

Reply:
The initial screen width should be 395 or more that's why it shows it not arranged. you have make the width of you screen a little larger and for larger screens it works fine.